bin/dfx-network-provider

```diff
@@ -30,14 +30,14 @@ get_home() {
 case "${FORMAT}" in
 url)
   if [[ "$DFX_NETWORK" == "local" ]]; then
-    REPLICA_PORT="$(dfx info replica-port 2>/dev/null || true)"
-    [[ "${REPLICA_PORT:-}" != "" ]] || {
+    WEBSERVER_PORT="$(dfx info webserver-port 2>/dev/null || true)"
+    [[ "${WEBSERVER_PORT:-}" != "" ]] || {
       # The above command can fail if not run in the same directory as the replica working directory.
-      DFX_EXEC_DIR="$(lsof -p $(pgrep pocket-ic) | grep cwd | awk '{print $NF}')"
+      DFX_EXEC_DIR="$(lsof -p "$(pgrep pocket-ic | head -1)" | grep cwd | awk '{print $NF}')"
```

Comment:
Is it important which process to pick?

Reply:
No, I expect all of them to be started by the same call to `dfx start` so they should all result in the same directory.